projects/drivers/axi_dac_core/axi_dac_core.c

```diff
@@ -796,16 +772,9 @@ int32_t axi_dac_set_buff(struct axi_dac *dac,
 	for(index = 0; index < buff_size; index += 2) {
 		data_i = (buff[index]);
 		data_q = (buff[index + 1] << 16);
-#ifdef ALTERA_PLATFORM
-		IOWR_32DIRECT(address, index * 2, data_i | data_q);
-#else
-		Xil_Out32(address + index * 2, data_i | data_q);
-#endif
-	}
 
-#ifndef ALTERA_PLATFORM
-	Xil_DCacheFlush();
```

Comment:
This functionality is completely removed. Don't we still need this?

Reply:
The project is working without the Cache flush. Keeping these functionalities will imply adding a special wrapper for this function to make the driver platform agnostic (besides axi_io read/write). So, I believe this subject is debatable.

Reply:
We need it - it should be moved to the main() function.